docs/user/howto/shared-deps.md

```diff
@@ -0,0 +1,165 @@
+# Working with shared dependencies
+
+Hypper charts understand the concept of shared dependency charts.
```

Comment:
```suggestion
Hypper charts provide a concept of shared dependency charts.
```

Reply:
:thinking:  mmh, I would say Hypper (or "the implementation") provides the concept of shared dependencies, but the charts "support" them?

Reply:
support sounds nice...

Reply:
fixed in https://github.com/rancher-sandbox/hypper/pull/131/commits/6bb8f6dddf819eab4ad0e589bd3285bf59853db0.